### PR TITLE
Multiple tables

### PR DESCRIPTION
**Pull request by Daphineatim**
### In this milestone, the following changes were implemented;

- Added some new tables 
- Add foreign key columns to existing animals table to model one-to-many relationships. 
- Joined to query the data.

tables1

tables2

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -8,3 +8,40 @@ SELECT * FROM animals WHERE weight_kg > 10.5;
...
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT name FROM animals WHERE escape_attempts = (SELECT MAX(escape_attempts) FROM animals);
+SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth>'01-01-1990' AND date_of_birth<'01-01-2000' GROUP BY species;
+
+-- Third project
+SELECT name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Melody Pond';
+SELECT animals.name FROM species JOIN animals ON species.id = animals.species_id WHERE species.name = 'Pokemon';
```
Comment (Nemwel-Boniface) [inline queries.sql]: - [ ] To correctly write this query, you are supposed to select  from the animals table, then joining the species table. Kindly consider making this change.

File: queries.sql [hunk 2]
```diff
@@ -8,3 +8,40 @@ SELECT * FROM animals WHERE weight_kg > 10.5;
...
+SELECT full_name, name FROM animals RIGHT JOIN owners ON animals.owner_id = owners.id;
```
Comment (Nemwel-Boniface) [inline queries.sql]: - [x] The screenshot provided in your PR description shows that you might be having some extra data which is not supposed to be there. The query results are wrong. Kindly recheck the data input and try again.

File: queries.sql [hunk 3]
```diff
@@ -8,3 +8,40 @@ SELECT * FROM animals WHERE weight_kg > 10.5;
...
+SELECT species.name, COUNT(*) FROM species JOIN animals ON species.id = animals.species_id GROUP BY species.name;
```
Comment (Nemwel-Boniface) [inline queries.sql]: - [x] The screenshot provided in your PR description shows that you might be having some extra data which is not supposed to be there. The query results are wrong. Kindly recheck the data input and try again.

File: queries.sql [hunk 4]
```diff
@@ -8,3 +8,40 @@ SELECT * FROM animals WHERE weight_kg > 10.5;
...
+SELECT name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Jennifer Orwell' AND animals.name LIKE '%mon';
```
Comment (Nemwel-Boniface) [inline queries.sql]: - [x] This query is wrong. You are supposed to list all **digimons** owned by **Jennifer Orwell.**. Kindly update it.

image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @Daphineatim       :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Good job working on your project so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Daphineatim) [reply queries.sql]: Thank you, I have worked on that.

File: queries.sql (hunk 4 above)
Comment (Daphineatim) [reply queries.sql]: I have made the changes. Thank you!

**Review by DeliceLydia — APPROVED** [review]
## Hi @Daphineatim, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**